metricflow/aggregation_properties.py

```diff
@@ -17,18 +17,29 @@ class AggregationType(ExtendedEnum):
     SUM_BOOLEAN = "sum_boolean"
     AVERAGE = "average"
 
-    # COUNT = "count" not yet implemented ... non-expansive as COUNT(COUNT(X), COUNT(Y)) != COUNT(COUNT(X ∪ Y))
+    COUNT = "count"  # is only expansive because it's transformed into a SUM agg during model transformation
```

Comment:
Sorry, I meant in the docstring on is_expansive. It's fine here but the docstring is better since people might access the property definition/documentation in their IDEs or whatever.